pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -465,17 +490,17 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 			cset := m.state.GetCPUSetOrDefault(string(pod.UID), container.Name)
 			if cset.IsEmpty() {
 				// NOTE: This should not happen outside of tests.
-				klog.V(2).InfoS("ReconcileState: skipping container; assigned cpuset is empty", "pod", klog.KObj(pod), "containerName", container.Name)
+				lh.V(2).Info("skipping container; assigned cpuset is empty")
```

Comment:
Ah, I see because we don't have `containerID` here?

With a lower verbosity makes sense to skip this entry.

Another candidate for `loggerV`?

Reply:
yes, I'll need to play a bit with `loggerV` to see how we can capture `containerID`. Will do for the next upload